contracts/p0/BasketHandler.sol

```diff
@@ -671,5 +715,5 @@ contract BasketHandlerP0 is ComponentP0, IBasketHandler {
      * variables without shifting down storage in the inheritance chain.
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      */
-    uint256[48] private __gap;
+    uint256[47] private __gap;
```

Comment:
We can honestly get rid of this gap in the p0 contracts

Reply:
Yes we should probably do that to avoid confusions.